docs/_snippets/patient.mdx

```diff
@@ -7,7 +7,7 @@
 <Info>
   You may provide a comma/space delimited string to specify multiple first and last names. For
   example, the following inputs would be equivalent: `"John,Jonathan"` & `"John Jonathan"` - these
-  would translate to `["John", "Jonathan"]`.
+  would translate to `["John", "Jonathan"]`. Use this when specifying nicknames and pre-marital names
```

Comment:
nit: can we just remove this? it adds extra words without much value - IMO `["John", "Jonathan"]` is explicit in the example that it can be used for short names and etc

Reply:
I found it unclear when I was going through but sure